### DISCUSSION TITLE
Einfaches Text-Display anschließen und Leistung anzeigen

### DISCUSSION THREAD

**Discussion by MichaMEG** [tbnobody/OpenDTU#352] [Ideas]
Könnte man evt. auf einem kleinen Text-Display, welches an die Pio-Pins angeschlossen wird, die aktuelle Leistung ausgegeben?

Danke & Grüße 

Michael

**Comment by raccoon0815** [comment]
https://www.az-delivery.de/blogs/azdelivery-blog-fur-arduino-und-raspberry-pi/e-paper-display-am-esp32-und-esp8266-teil1

**Comment by HacksBugsAndRockAndRoll** [comment]
Ich habe für die Verwandtschaft ohne SmartHome Infrastruktur etwas mit einer 7Segment Anzeige zusammengesteckt. Vielleicht hilft das bei der Ideenfindung :).
![image](https://user-images.githubusercontent.com/61069044/202033638-acc29f56-d3b0-45b2-85d4-0b46a0b6bbf2.png)

Code Anpassung für die Verwendung mit  "MAX7219 8 Bit digital Segment" Anzeige.
https://github.com/HacksBugsAndRockAndRoll/OpenDTU/tree/local/7segment-display/src

**Reply by scorpi11** [reply]
Sowas in der Art habe ich auch noch vor, evtl. mit einem OLED-Display. Aber die sind leider alle sehr klein.

Bisher lasse ich nur die blaue LED blinken wenn der Wechselrichter aktiv ist, damit ich erkenne, ob Daten per Web-Interface abgerufen werden können.
